### PR TITLE
Revision of 3D Secure 2.0

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/methods/credit-and-debit-cards/3d-secure/_index.md [hunk 1]
```diff
@@ -59,7 +57,9 @@ To help you optimize conversion and manage risk, MultiSafepay supports exemption
 
 ### Merchant-initiated transactions
 
-MultiSafepay supports merchant-initiated transactions (MIT) as [recurring payments](/features/recurring-payments).
+Merchant-initiated transactions are [recurring payments](/features/recurring-payments) initiated by the merchant on behalf of the customer, based on a prior agreement. Examples of merchant-initiated transactions include subscription payments and automatic balance top-ups. 
+
+Merchant-initiated transactions do not require 3D Secure 2.0 authentication.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:63]:
```suggestion

### Mail Order/Telephone Order (MOTO)
 
For [MOTO](https://docs.multisafepay.com/features/moto/) payments, the customer provides their card details to the merchant over the phone or by email. These payments do not require 3D Secure 2.0 authentication. 
```

File: content/payments/methods/credit-and-debit-cards/3d-secure/_index.md [hunk 2]
```diff
@@ -59,8 +57,13 @@ To help you optimize conversion and manage risk, MultiSafepay supports exemption
 
 ### Merchant-initiated transactions
 
-MultiSafepay supports merchant-initiated transactions (MIT) as [recurring payments](/features/recurring-payments).
+Merchant-initiated transactions are [recurring payments](/features/recurring-payments) initiated by the merchant on behalf of the customer, based on a prior agreement. Examples of merchant-initiated transactions include subscription payments and automatic balance top-ups.
```
Comment (FrancescaNCook) [inline content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:60]:
```suggestion
Merchant-initiated transactions (MIT) are [recurring payments](/features/recurring-payments) initiated by the merchant on the customer's behalf, based on a prior agreement. Examples include subscription payments and automatic balance top-ups. 
```

File: content/payments/methods/credit-and-debit-cards/3d-secure/_index.md [hunk 3]
```diff
@@ -59,8 +57,13 @@ To help you optimize conversion and manage risk, MultiSafepay supports exemption
 
 ### Merchant-initiated transactions
 
-MultiSafepay supports merchant-initiated transactions (MIT) as [recurring payments](/features/recurring-payments).
+Merchant-initiated transactions (MIT) are [recurring payments](/features/recurring-payments) initiated by the merchant on the customer's behalf, based on a prior agreement. Examples include subscription payments and automatic balance top-ups. 
+
+Merchant-initiated transactions do not require 3D Secure 2.0 authentication.
```
Comment (FrancescaNCook) [inline content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:62]:
```suggestion
```

Comment (FrancescaNCook) [reply content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:62]: This is already under the heading of exemptions.

File: content/payments/methods/credit-and-debit-cards/3d-secure/_index.md [hunk 4]
```diff
@@ -59,8 +57,12 @@ To help you optimize conversion and manage risk, MultiSafepay supports exemption
 
 ### Merchant-initiated transactions
 
-MultiSafepay supports merchant-initiated transactions (MIT) as [recurring payments](/features/recurring-payments).
+Merchant-initiated transactions (MIT) are [recurring payments](/features/recurring-payments) initiated by the merchant on the customer's behalf, based on a prior agreement. Examples include subscription payments and automatic balance top-ups. 
+
 
+### Mail Order/Telephone Order (MOTO)
+ 
+For [MOTO](https://docs.multisafepay.com/features/moto/) payments, the customer provides their card details to the merchant over the phone or by email. These payments do not require 3D Secure 2.0 authentication.
```
Comment (FrancescaNCook) [inline content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:65]:
```suggestion
For [MOTO](https://docs.multisafepay.com/features/moto/) payments, the customer gives you their card details by phone or email. 3DS2 authentication is **not required**. 
```

File: content/payments/methods/credit-and-debit-cards/3d-secure/_index.md [hunk 5]
```diff
@@ -59,8 +57,12 @@ To help you optimize conversion and manage risk, MultiSafepay supports exemption
 
 ### Merchant-initiated transactions
 
-MultiSafepay supports merchant-initiated transactions (MIT) as [recurring payments](/features/recurring-payments).
+Merchant-initiated transactions (MIT) are [recurring payments](/features/recurring-payments) initiated by the merchant on the customer's behalf, based on a prior agreement. Examples include subscription payments and automatic balance top-ups.
```
Comment (FrancescaNCook) [inline content/payments/methods/credit-and-debit-cards/3d-secure/_index.md:60]:
```suggestion
Merchant-initiated transactions (MIT) are [recurring payments](/features/recurring-payments) you initiate on the customer's behalf, based on a prior agreement, e.g. subscriptions, automatic balance top-ups. 3DS2 authentication is **not** required.
```

**Review by FrancescaNCook — APPROVED** [review]

**Pull request merged**